OpenRA.Game/Actor.cs

```diff
@@ -51,6 +51,7 @@ internal struct SyncHash
 		public IEffectiveOwner EffectiveOwner { get; private set; }
 		public IOccupySpace OccupiesSpace { get; private set; }
 		public ITargetable[] Targetables { get; private set; }
+		public ITargetablePositions[] TargetablePositionTraits { get; private set; }
```

Comment:
Maybe name it `TargetablePositions` for consistancy with the other cached traits?